### PR TITLE
For getting is-a.dev domain for portfolio website

### PR DESCRIPTION
**Pull request by ajachintya**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

File: domains/achintya.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "ajachintya",
+    "email": "ajachintyajaiswal@gmail.com",
+  },
+  "record": {
+    "URL": "portfolio-app-production.up.railway.app"
```
Comment (WilliamDavidHarrison) [inline domains/achintya.json:7]:
```suggestion
    "URL": "https://portfolio-app-production.up.railway.app"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Didn't mean to approve

**Review by ajachintya — COMMENTED** [review]
added "https://" in the url

**Review by wdhdev — DISMISSED** [review]

**Comment by ajachintya** [comment]
How long can it take to get approved by a maintainer?

**Comment by WilliamDavidHarrison** [comment]
> How long can it take to get approved by a maintainer?

Hopefully not too long.

@andrewstech

**Comment by ajachintya** [comment]
Could it be possible to speed up the review process? Thank you in advance :)

**Comment by WilliamDavidHarrison** [comment]
> Could it be possible to speed up the review process? Thank you in advance :)

Sorry that is not possible.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**